setup.py

```diff
@@ -26,7 +26,8 @@
     install_requires=[
         'requests~=2.28.0',
         'PyYAML~=6.0',
-        'databricks_cli==0.17.0'
+        'databricks_cli==0.17.1',
+        'rich==12.5.1'
```

Comment:
Why restrict only to these versions? Maybe use a compatible version specifier

Reply:
Yes. Using =~